### DISCUSSION TITLE
Taskbar icon combining stopped working

### DISCUSSION THREAD

**Discussion by justinsutw** [valinet/ExplorerPatcher#1308] [General]
It happened to me yesterday. Did anyone else notice that?

**Comment by PabloGirol** [comment]
Could you send a screen capture as an example?

**Reply by justinsutw** [reply]
This is what I have
<img width="793" alt="image" src="https://user-images.githubusercontent.com/19938798/233693318-c4c94b1f-a431-4639-90ba-79b1a8b37e2c.png">

With the icon labels no longer visible, I am expecting the icons from the same program to combine.
